packages/components/organisms/f-checkout/src/components/Selector.vue

```diff
@@ -40,36 +40,48 @@ export default {
         * Returns an array of formatted dates to display as options in a dropdown
         */
         fulfilmentTimes () {
-            const times = this.availableFulfilment.isAsapAvailable
-                ? [this.$t('asapFulfilmentOption')]
-                : [];
-
+            const times = [];
             this.availableFulfilment.times.forEach(time => {
                 const formattedTime = this.$d(new Date(time.from), 'short');
 
-                times.push(formattedTime);
+                times.push({
+                    text: formattedTime,
```

Comment:
The latest f-form-field has support for different text and value fields, so we can show something like "Wednesday 12:00" in the dropdown but when we submit to the API it will be in ISO format.